installing/install_config/installing-restricted-networks-preparations.adoc

```diff
@@ -5,7 +5,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. Installations on a restricted network are supported on only infrastructure that you provision, not infrastructure that the installer provisions. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. Installations on a restricted network are supported on only infrastructure that you provision, not infrastructure that the installer provisions.
+
+You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content. You will have to block your cluster from reaching the upstream registry in an unrestricted network. If your cluster can reach the upstream registry it will still prefer it over your local mirror.
```

Comment:
```suggestion
You can also create a mirror that contains the required container images in unrestricted networks to ensure your clusters use only container images that satisfy your organizational controls on external content. You must block your cluster from reaching the upstream registry in an unrestricted network. If your cluster can reach the upstream registry, it uses the images from that registry instead of the local mirror.
```

Reply:
Will you remind me what specific separate step users need to take to block the cluster from accessing the upstream registry?

Reply:
> Will you remind me what specific separate step users need to take to block the cluster from accessing the upstream registry?

That is up to the user, I blacklist quay.io in my dns server so it doesn't resolve. But I'd guess you can also block the IPs in firewall or similar :)

Reply:
also keep in mind that there are different targets that need to be blocked depending on your flavour (okd/openshift)